uiowa/uiowa
# Feature: Added menu per role module

Resolves https://github.com/uiowa/uiowa/issues/3188. 

# Work remaining to be done in this pr:

- [x] Only show checkboxes for admins (Current permissions overriden by `Menu Admin per Menu` module)
**or** 
- [x] Only show one checkbox to hide for Anonymous users (For all roles except admin).
- [ ] Fix superfish hover/arrow issues

# How to test
- `composer install`
- `blt ds --site=sandbox.uiowa.edu`
- As an admin, you can edit any menu item and hide the item to anonymous users. As an admin you shouldn't see any other options. Save and open the site in an unauthenticated session.
- As a webmaster, editor, etc. You can't see the hide functionality on the menu item but you can still see the menu link within admin/structure/menu. On the hidden menu item you should see a message saying that the menu item you previously set is hidden.
- You can add the `administer menu link visibility` permission to editor, webmaster, publisher and they should then see the hide functionality. This permission is not set as a default for any role. This functionality is limited to admins and site/feature splits right now.
- Even as an admin, you shouldn't see the "show" functionality.
